HTML/index.html

```diff
@@ -33,15 +33,13 @@
       </div>
 
       <div class="row">
-        <div class="col-sm-8">
-          <img src="math.png" alt="damn damn" width=100% height=40%/>
+        <div class="price">
```

Comment:
`col-sm-8` can be added back into the class declarations: `class="price col-sm-8"`

Same can be done below in the `info` div